003

```diff
@@ -1,23 +1,35 @@
-The goal of KIP 003 is creat a staking incentive.  We will copy some of the same principles implementd on the AAVE chain.
-First off a general summary of roughly how he AAVE network bootstapped their staking and some of the loose numbers in the reserve.
-AAVE has a total supply of 16 milling AAVE.
-AAVE emissions rate is 1,100 AAVE per day emitted for staking.
-The supply is fixed.
-This is only emissions source, and they come from the ecosystem reserve.
-the ecosystem reserve is designed to last for only a couple of years currently.
-The initial ecosystem reserve amount had 3 million AAVE at the time of toke inception.
+# KIP 003 - Protocol Staking Incentives 
+
+### The goal of KIP 003 is create a staking incentive.  Our intention is to borrow from already successful protocol staking mechanisms.
+
+We're going to use AAVE as a case study because their safety module is the exact kind of incentive system we want to create.
+What makes this an effective incentive system is that it encourages participation in securing the protocol and that its simple and transparent.  
+
+Firstly, let's get a general sense of how AAVE network bootstapped their staking and some of the loose numbers in the reserve.
+
+- AAVE has a total supply of 16 million AAVE tokens so the chosen AAVE emissions rate for the protocol staking incentives is 1,100 AAVE per day emitted.
```

Comment:
Making the emission percentage .006875% of Total Reserve released per day.  We would like to use this percentage also for our emission rate.